src/treeChooser/treeChooser.controller.js

```diff
@@ -26,7 +26,10 @@ function TreeChooserController(
 
     // Add event listener to determine when user clicks outside of tree chooser
     if (!vm.disableClick) {
-      $window.addEventListener('click', vm.closeFromClick);
+      //wrap to $timeout to avoid immediate call because of current click event
```

Comment:
I don't follow this. This is detecting the click event that caused this to be added?